README.md

```diff
@@ -150,7 +150,14 @@ The `OR` operator works like this:
 users.where(users[:name].eq('bob').or(users[:age].lt(25)))
 ```
 
-The `AND` operator behaves similarly.
+The `AND` operator behaves similarly. Here is an example of the `DISTINCT` operator, which as of v6 is also chainable:
```

Comment:
I think we can safely remove any version reference from the README since the readme is attached to the current version that is 8.0 now